src/data/fintype/basic.lean

```diff
@@ -170,6 +170,14 @@ lemma univ_filter_mem_range (f : α → β) [fintype β]
   finset.univ.filter (λ y, y ∈ set.range f) = finset.univ.image f :=
 univ_filter_exists f
 
+/-- A special case of `finset.sup_eq_supr` that omits the useless `x ∈ univ` binder. -/
+lemma sup_univ_eq_supr [complete_lattice β] (f : α → β) : finset.univ.sup f = supr f :=
```

Comment:
Should these be simp-lemmas?

Reply:
`sup_eq_supr` isn't, so I'd say no. I expect there are cases where the finiteness of `sup` makes it easier to work with than alternatives, meaning that if you put it in your goal you want it to remain there.